go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	go.opencensus.io v0.22.0
 	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
 	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 // indirect
+	google.golang.org/api v0.4.0
```

Comment:
Any idea how this library was required ?

Reply:
I tried to use `testing` in this module and switched to [official testing](https://golang.org/pkg/testing/). Remove this